pkg/controller/quota/quotaforestmanager/qm_lib_backend_with_quotasubt_mgr.go

```diff
@@ -127,25 +131,24 @@ func NewQuotaManager(dispatchedAWDemands map[string]*clusterstateapi.Resource, d
 	klog.V(10).Infof("[NewQuotaManager] Before initialization QuotaSubtree informer - %s", qm.quotaManagerBackend.String())
 
 	// Create a resource plan manager
-	qm.quotaSubtreeManager, _ = qstmanager.NewQuotaSubtreeManager(config, qm.quotaManagerBackend)
+	qm.quotaSubtreeManager, err = qstmanager.NewQuotaSubtreeManager(config, qm.quotaManagerBackend)
+	if err != nil {
+		klog.Errorf("[NewQuotaManager] failed to update the instantiate new quota subtree manager, err=%#v", err)
```

Comment:
Nit: clarify the log message, should it say `failed to instantiate new quota subtree manager` ?

Reply:
Done !